test/schema/collection.js

```diff
@@ -49,9 +36,7 @@ describe("Collections", () => {
     it("returns artworks for a collection", () => {
       const artworksPath = resolve("test", "fixtures", "gravity", "artworks_array.json")
       const artworks = JSON.parse(readFileSync(artworksPath, "utf8"))
-      gravity
-        .withArgs("collection/saved-artwork/artworks", { size: 10, offset: 0, total_count: true, user_id: "user-42" })
-        .returns(Promise.resolve({ body: artworks, headers: { "x-total-count": 10 } }))
+      gravityData = { body: artworks, headers: { "x-total-count": 10 } }
```

Comment:
This currently loses the "this API + this args" inside the definition of the test. This is optional, I can bring it back?